docs/examples/perf_analyzer/README.md

```diff
@@ -0,0 +1,46 @@
+# perf_analyzer example with DALI image decoder
+
+
+## Run the example
+
+### tritonserver
+
+To run the tritonserver, first you need to serialize DALI pipeline and put it inside model repo.
+
+`setup_perf_analyzer_example.sh` is a convenience script, that automatizes setting up.
+Provided you have DALI wheel installed in your system, you can just call `sh ./setup_perf_analyzer_example.sh`
+
+When you have your model repo set up, you can run `tritonserver`. Be sure to fill `<path to model repo>` properly:
+
+    docker run -it --rm --shm-size=1g --ulimit memlock=-1 --gpus all --ulimit stack=67108864 -p8000:8000 -p8001:8001 -p8002:8002 --privileged -v <path to model repo>:/models nvcr.io/nvidia/tritonserver:XX.YY-py3 tritonserver --model-repository=/models
```

Comment:
Can you provide any working example that the user can just copy-past and run (example values for XX.YY and <path to model repo>)?

Reply:
I don't really know, what `-v <path to model repo>:/models` user will have. As far as `XX.YY` goes, I used it so that this doc won't need to be updated every time new `tritonserver` version is released.

Reply:
I was asking for a simple example that anyone can just copy-past and run just to check how it works before moving to the real profiling.
Also I don't really know what `<path to model repo>:/models` should be. So even a simple example would provide a lot of info.

Reply:
To run this example, user has to set up the model repository quasi-manually (using the `setup` script, it's mentioned earlier in the README). Therefore I wouldn't like to impose any model_repository path here. I've put a short explanation there, following @jantonguirao suggestion. Is that ok?

Reply:
Sure. I'm just saying that I don't know TRION much and I cannot run this easily on my own to verify.